babel.config.js

```diff
@@ -1,3 +1,6 @@
+const PKG_PATH = process.cwd();
```

Comment:
not `.babelrc`?

Reply:
Looks like .babelrc.js doesn't get resolved.
 .babelrc -> json, .babelrc.js -> not working